test/read_write_test.py

```diff
@@ -0,0 +1,76 @@
+# Databricks notebook source
+# imports
+from data_io.connection import build_connection
+from urlpath import URL
+from pyspark.sql import SparkSession
+from pyspark.dbutils import DBUtils
+"""
+unittest does note work in databricks (https://stackoverflow.com/questions/67314705/databricks-python-unit-test-error-help-needed),
+so we will go with nutter (https://github.com/microsoft/nutter)
+"""
+from runtime.nutterfixture import NutterFixture, tag
+
+class DataioReadWrite(NutterFixture):
+  
+  """
+  For this unittest we will kill 2 birds with one stone.
+  where going to wirte a dummy dataframe to redshifts, S3, and oracle.
+  Than read it back and test it is the same as the dummy dataframe
+  """
+  
+  def __init__(self):
+    #our dummy dataframe
+    self.df = spark.createDataFrame([
+                (1, 'foo'),
+                (2, 'bar')],
+                ['id', 'label']
+                  )
+    #s3 test location
+    self.s3_url = dbutils.secrets.get(scope= 'data_io.test', key = "s3_test")
```

Comment:
parameters do not have spaces before/after operator.

Please run through Linter.